FASTDOOM/i_ibm.c

```diff
@@ -371,6 +376,266 @@ const byte colors[48] = {
     0x3F, 0x3F, 0x3F};
 #endif
 
+#if defined(MODE_CGA136) || defined(MODE_VGA136)
+const byte colors[3 * 256] = {
```

Comment:
Wait, there are duplicate colors in this table. It should have at most 136 elements.